cfg/constants.go

```diff
@@ -33,3 +33,8 @@ const (
 	// ExperimentalMetadataPrefetchOnMountAsynchronous is the prefetch-mode where mounting is marked complete once prefetch has started.
 	ExperimentalMetadataPrefetchOnMountAsynchronous = "async"
 )
+
+const (
+	// MaxSequentialReadSizeMb is the max value supported by sequential-read-size-mb flag.
+	MaxSequentialReadSizeMB = 1024
```

Comment:
Does it really have to be exported? All the usages seem to be confined to the same package.